crates/release_plz_core/src/next_ver.rs

```diff
@@ -905,6 +906,12 @@ impl Updater<'_> {
         loop {
             let current_commit_message = repository.current_commit_message()?;
             let current_commit_hash = repository.current_commit_hash()?;
+
+            // Check if files changed in git commit belong to the current package.
+            // This is required because a package can contain another package in a subdirectory.
+            let are_changed_files_in_pkg =
```

Comment:
I think we can just compute the result here

Reply:
By using a closure the function is executed only if needed. I.e. If we reach the else if cases.

Instead, if we compute the result there, the function is always executed.

By using a closure we save compute. What do you think?

Reply:
Thats true but in most cases the closure will be executed. But besides that i tested it and everything is working properly